### DISCUSSION TITLE
CORS not letting images display on flutter web even with  headers config in vercel.json

### DISCUSSION THREAD

**Discussion by Aarush-Acharya** [vercel/community#1188] [Help]
### Summary

The image request does not work when any random image URL is copied from web 
<img width="1438" alt="Screenshot 2022-12-24 at 10 51 29 PM" src="https://user-images.githubusercontent.com/92685647/209445861-c2379a3d-6d1c-4839-82f9-7e12a837967c.png">

For an instance: https://builtin.com/sites/www.builtin.com/files/styles/og/public/2022-09/blockchain.png
The web does not allow the request to execute

Note: Working in dialog flow and with flutter web building chat bot and needed image as a response 

But it works when the image is sent to someone on discord and discord starts hosting it 
discord image link:
https://images-ext-2.discordapp.net/external/CsJLhAfbBc6tNkfb4LUUUwjsP
Z9HYlhvkA6IWWg5y4/https/builtin.com/sites/www.builtin.com/files/styles/og/public/2022-09/blockchain.png

<img width="1440" alt="Screenshot 2022-12-24 at 10 53 08 PM" src="https://user-images.githubusercontent.com/92685647/209445907-23e51e26-6055-4866-ade8-640126dabe2e.png">

This is my vercel.json file

```json
{
    "headers": [
      {
        "source": "/(.*)",
        "headers": [
          { "key": "Access-Control-Allow-Credentials", "value": "true"},
          { "key": "Access-Control-Allow-Origin", "value": "*"},
          { "key": "Access-Control-Allow-Methods", "value": "GET,OPTIONS,PATCH,DELETE,POST,PUT"},
          { "key": "Access-Control-Allow-Headers", "value": "X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version"}
        ]
      }
    ]
}
``` 

### Example

https://chabu.vercel.app/#/

**Comment by hamzaaouni** [comment]
you can add the following configuration to your vercel.json file:

```
{
  "headers": [
    {
      "source": "/(.*)",
      "headers": [
        {
          "key": "Access-Control-Allow-Origin",
          "value": "*"
        }
      ]
    }
  ]
}
```
